.github/workflows/ci-go.yml

```diff
@@ -11,6 +11,9 @@ on:
 permissions:
   contents: read
 
+env:
+  GOFLAGS: -tags=aro,containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper
```

Comment:
I'm concerned about excluding things this way, because we aren't fully testing what we are deploying, no?

Reply:
@cblecker `containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper` is for the containers.io container storage backends which we don't use (because we're not running as a container daemon) and introduce dependencies which are quite annoying.